_posts/meat/2018-08-05-pla-rad-prik.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Pla Rad Prik
+category: meat
+---
+
+[Pla Rad Prik][pla_rad_prik] is a Thai dish: whole-fried, meaty white fish with
+chili sauce, usually a centerpiece entree. It's pretty common to find red
+snapper (delicious!) in Thai restaurants in the US; farmed or less over-fished
+alternatives like grouper, pompano, white perch, haddock, etc. probably work as
+well. Ideally, whatever fish is sized appropriately for your wok.
+
+- For the sauce, food process:
+  * 1/2 cup - garlic cloves
+  * 1/2 cup - small, red Thai chilies (de-seeded)
+
+  Also set aside:
+  * 1/2 cup - tamarind paste (prepared from pulp, if possible).
```

Comment:
what do you mean by 'prepared from pulp'? can you link to photos of the pulpy kind and the liquidy kind, this was not obvious to me until recently.